### PR TITLE
Improve Risk of Rain 2 docs

### PR DESCRIPTION
**Pull request by mDuo13**
Having played Risk of Rain 2 with a few people new to Archipelago, there are several pitfalls and misunderstandings that seem to come up a lot, so I have attempted to clarify them.

- How multiplayer works
- Victory conditions (this is updated assuming the fixes from #766 are in effect)
    - Seems pretty common for people to think that clearing all the location checks is required to win, maybe because it appears in the objective list.
- Compatibility with other mods
- The fact that you need to change "Item Weight Presets" to "No" or else your adjustments to the item frequencies won't apply

### CODE REVIEW

File: worlds/ror2/Options.py [hunk 1]
```diff
@@ -123,14 +123,14 @@ class ItemPoolPresetToggle(DefaultOnToggle):
     display_name = "Item Weight Presets"
```
Comment (alwaysintreble) [inline worlds/ror2/Options.py:123]: Maybe you could change the display name of this option as well to help make it clearer?

File: worlds/ror2/docs/en_Risk of Rain 2.md [hunk 2]
```diff
@@ -31,13 +57,34 @@ in-game item of that tier will appear in the Risk of Rain player's inventory. If
 the player already has an equipment item equipped then the _item that was equipped_ will be dropped on the ground and _
 the new equipment_ will take it's place. (If you want the old one back, pick it up.)
 
+### How many items are there?
+
+Since a Risk of Rain 2 run can go on indefinitely, you have to configure how many collectible items (also known as
+"checks") the game has for purposes of Archipelago when you set up a multiworld. You can configure anywhere from **10
+to 500** items. The number of items will be randomized between all players, so you may want to adjust the number and
```
Comment (alwaysintreble) [inline worlds/ror2/docs/en_Risk of Rain 2.md:64]: Decision has finally been made to reduce this number to 250 which I just finished doing here https://github.com/ArchipelagoMW/Archipelago/pull/776

**Review by alwaysintreble — COMMENTED** [review]
2 minor suggestions but rest of it lgtm

File: worlds/ror2/docs/en_Risk of Rain 2.md (hunk 2 above)
Comment (Ijwu) [inline worlds/ror2/docs/en_Risk of Rain 2.md:64]:
```suggestion
to 250** items. The number of items will be randomized between all players, so you may want to adjust the number and
```

**Review by Ijwu — APPROVED** [review]

**Pull request merged**